fundamentals/01_data_structures.md

```diff
@@ -1,5 +1,69 @@
 # Data Structures
 
+Multi-dimensional (a.k.a. N-dimensional, ND) arrays (sometimes called “tensors”)
+are an essential part of computational science. They are encountered in a wide
+range of fields, including physics, astronomy, geoscience, bioinformatics,
+engineering, finance, and deep learning. In Python, [NumPy](https://numpy.org/)
+provides the fundamental data structure and API for working with raw ND arrays.
+However, real-world datasets are usually more than just raw numbers; they have
+labels which encode information about how the array values map to locations in
+space, time, etc.
+
+The N-dimensional nature of xarray’s data structures makes it suitable for
+dealing with multi-dimensional scientific data, and its use of dimension names
+instead of axis labels (`dim='time'` instead of `axis=0`) makes such arrays much
+more manageable than the raw numpy ndarray: with xarray, you don’t need to keep
```

Comment:
```suggestion
The N-dimensional nature of Xarray’s data structures makes it suitable for
dealing with multi-dimensional scientific data, and its use of dimension names
instead of axis labels (`dim='time'` instead of `axis=0`) makes such arrays much
more manageable than the raw NumPy ndarray: with Xarray, you don’t need to keep
```